rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-05-15
+nightly-2020-08-26
```

Comment:
Interesting, why newest Rust is needed? Thought Wasmer promised to work with the stable Rust.

Reply:
wasmer 1.0 doesn't compile with 0515, we can see this by a quick test:
```
cargo new testwasmer
cd testwasmer
# add wasmer 1.0 to Cargo.toml
echo  nightly-2020-05-15 > rust-toolchain
cargo build
```
On my laptop, i saw 1517 compile errors :)
```

Reply:
Note: Wasmer now also supports Rust stable (1.45+)

Reply:
Thanks! will use stable